app/src/validator/user.js

```diff
@@ -0,0 +1,36 @@
+const { validate, Joi } = require('express-validation');
+
+const schema = {
+  searchUsers: {
+    query: Joi.object({
+      userId: Joi.array().items(Joi.string().guid({
```

Comment:
We're using the mixedQueryToArray() method to turn one or more values for parameters (eg: userId and identityId) into an array, but they can be provided as a string in the query.
eg: /api/user?userId=098090-324-24-23-4234 should be valid.